bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/productlist/CategoryRetriever.java

```diff
@@ -17,48 +17,13 @@
 import com.adobe.cq.commerce.core.components.models.retriever.AbstractCategoryRetriever;
 import com.adobe.cq.commerce.magento.graphql.CategoryTreeQuery;
 import com.adobe.cq.commerce.magento.graphql.CategoryTreeQueryDefinition;
-import com.adobe.cq.commerce.magento.graphql.ProductInterfaceQuery;
-import com.adobe.cq.commerce.magento.graphql.ProductInterfaceQueryDefinition;
-import com.adobe.cq.commerce.magento.graphql.ProductPriceQueryDefinition;
 
 class CategoryRetriever extends AbstractCategoryRetriever {
 
     CategoryRetriever(MagentoGraphqlClient client) {
         super(client);
     }
 
-    private ProductPriceQueryDefinition generatePriceQuery() {
```

Comment:
This class no longer is responsible for generating product searches, this is all being handled in the search service